SpatialFullConvolution.lua

```diff
@@ -8,43 +8,20 @@ autotunerCache[1] = {} -- forward
 autotunerCache[2] = {} -- backwardFilter
 autotunerCache[3] = {} -- backwardData
 
-function SpatialFullConvolution:__init(nInputPlane, nOutputPlane,
-                            kW, kH, dW, dH, padW, padH, groups)
-    local delayedReset = self.reset
-    self.reset = function() end
-    parent.__init(self, nInputPlane, nOutputPlane, kW, kH, dW, dH)
-    self.reset = delayedReset
-    self.padW = padW or 0
-    self.padH = padH or 0
-    self.groups = groups or 1
-    assert(nInputPlane % self.groups == 0,
-           'nInputPlane should be divisible by nGroups')
-    assert(nOutputPlane % self.groups == 0,
-           'nOutputPlane should be divisible by nGroups')
-    self.weight = torch.Tensor(nOutputPlane, nInputPlane/self.groups, kH, kW)
-    self.gradWeight = torch.Tensor(nOutputPlane, nInputPlane/self.groups, kH, kW)
-    self:reset()
-    -- should nil for serialization, the reset will still work
-    self.reset = nil
-    self.iSize = torch.LongStorage(4):fill(0)
-end
-
```

Comment:
this removes groups support ! that means that we will not be able to run any alexnet networks in torch.